### PR TITLE
Add missing image tag, to make same as other repos

### PR DESCRIPTION
**Pull request by redhug1**
### What

As per title

### How to review

Visual inspection (and compare to other repos, should be the same)

### Who can review

Anyone.

### CODE REVIEW

File: Dockerfile.concourse [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM onsdigital/dp-concourse-tools-ubuntu
+FROM onsdigital/dp-concourse-tools-ubuntu:ubuntu20.4-rc.1
```
Comment (dhee-tree) [inline Dockerfile.concourse]: Majority of the other repo uses `ubuntu-20:ubuntu20.4-rc.1`

Is this much of a difference?

Comment (redhug1) [reply Dockerfile.concourse]: oops, my mistake, will fix now.
thanks

Comment (redhug1) [reply Dockerfile.concourse]: done

**Review by dhee-tree — APPROVED** [review]
LGTM

**Pull request merged**